plugin/ruby.vim

```diff
@@ -172,5 +176,16 @@ class RubyTest
     cmd += test_command
     Vim.command("call VimuxRunCommand(\"#{cmd}\")")
   end
+
+  private
+
+  def get_locked_rspec_version
+    matches = get_locked_gem_version('rspec-core').scan(RSPEC_VERSION_REGEX) || get_locked_gem_version('rspec').scan(RSPEC_VERSION_REGEX)
+    matches.flatten.last
+  end
+
+  def get_locked_gem_version(gem_name)
+    `cat Gemfile.lock | grep '#{gem_name} (' | grep -v '[~,<,>,=]' | awk -F'[()]' '{print $2}'`
```

Comment:
Minor nitpick, but you can do `grep pattern file` instead of `cat file | grep pattern`.